integration-tests/manager-integration-tests/manager-integration-tests-v1/pom.xml

```diff
@@ -32,7 +32,7 @@
       <dependency>
         <groupId>com.squareup.okhttp3</groupId>
         <artifactId>okhttp</artifactId>
-        <version>4.9.2</version>
+        <version>4.10.0</version>
```

Comment:
What about move version in a property defined in the build-parent? Keep here the declaration but have the version declared in a single place.
Wdyt?

Reply:
Apparently these integration tests need a specific `okhttp` version (there is a comment `  <!-- Using custom version of okhttp due to dependency misalignment between RHOSE and TNB. -->`). So I wonder if we should keep the version here instead of in the build-parent :/

Reply:
Mmm this sounds a bit strange to me but it is unrelated to the PR so fine for me to proceed :)